tutorials/text_processing/WFST_Tutorial.ipynb

```diff
@@ -0,0 +1,8368 @@
+   },
+   "outputs": [],
+   "source": [
+    "LANGUAGE= \"fr\" # Change this to your desired language\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "_PAyEPhaFpCD",
+    "outputId": "23d034d2-8c93-4e8b-e3ce-5ba9e870f82d"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[Errno 2] No such file or directory: 'nemo_text_processing/inverse_text_normalization/'\n",
+      "/content/NeMo/nemo_text_processing/inverse_text_normalization/fr\n",
+      "/content/NeMo/nemo_text_processing/inverse_text_normalization/fr/fr\n",
+      "/content/NeMo/nemo_text_processing/inverse_text_normalization/fr/fr\n",
+      "data  taggers  verbalizers\n"
+     ]
+    }
+   ],
+   "source": [
+    "%cd nemo_text_processing/inverse_text_normalization/\n",
+    "!mkdir {LANGUAGE}\n",
+    "!mkdir \"{LANGUAGE}/taggers\"\n",
+    "!mkdir \"{LANGUAGE}/verbalizers\"\n",
+    "!mkdir \"{LANGUAGE}/data\"\n",
+    "%cd {LANGUAGE}\n",
+    "!pwd && ls"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "O1vfz-bUFpwz"
+   },
+   "source": [
+    "## GraphFst and Utils\n",
+    "\n",
+    "All WFSTs deployed in NeMo inherit from the `GraphFst` class in `nemo_text_processing.text_normalization.en.graph_utils`.\n",
+    "While in most cases you can simply import from the pre-existing `graph_utils` you may find it helpful for deployment to keep a copy \n",
+    "in your working directory for language specific edits. (For example, we will be utilizing `nemo_text_processing.text_normalization.en.graph_utils` as it provides some language-specific helper functions.)\n",
+    "\n",
+    "You may also wish to keep a copy of `utils.py` (found in each language folder in nemo_text_processing/inverse_text_normalization)\n",
+    "in your working directory to assist with pathing. (Make sure to adjust the imports towards your language.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "3OME84EmOQ4h",
+    "outputId": "6eea17f9-aae9-4176-ae35-3d1f0e94b4ea"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/content/NeMo\n"
+     ]
+    }
+   ],
+   "source": [
+    "!cp ../../text_normalization/en/graph_utils.py .\n",
+    "!cp ../../text_normalization/en/utils.py .\n",
+    "% cd ../../.."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "BzqQY41Qfk16"
+   },
+   "source": [
+    "## WFST Classes\n",
+    "\n",
+    "NeMo ITN's base languages currently support only the following semiotic classes to permit integration with Sparrowhawk deployment. \n",
+    "\n",
+    "- CARDINAL\n",
+    "- DATE\n",
+    "- DECIMAL\n",
+    "- FRACTION\n",
+    "- MEASURE\n",
+    "- MONEY\n",
+    "- NAME\n",
```

Comment:
maybe change this to PLAIN to be consistent with Google taxonomy?

Reply:
Won't this be confusing since the Taggers and Verbalizers consistently use Name?